build.bash

```diff
@@ -0,0 +1,80 @@
+# Copyright 2024 Adevinta
+
+if [[ -z $LAVA_VERSION ]]; then
+	echo 'error: missing env var LAVA_VERSION' >&2
+	exit 2
+fi
+
+check_status (){
+	local status=$1
+	local error_msg=$2
+	if [[ $status -ne 0 ]]; then
+		echo $error_msg
+		exit $status
+	fi
+}
+
+fix_headers (){
+	local file=$1
+	sed -i "s/^#/##/" ${file}
+	check_status $? "error: could not fix headers in file ${file}"
+
+}
+
+add_main_header(){
+	local file=$1
+	local header=$2
+	sed -i "1s/^/# ${header}\n/" ${file}
+	check_status $? "error: could not add main header ${header} to file ${file}"
+}
+
+GH_DOWNLOAD_URL_FMT='https://github.com/adevinta/lava/releases/download/%s/lava_linux_amd64.tar.gz'
+GH_DOWNLOAD_URL_LATEST='https://github.com/adevinta/lava/releases/latest/download/lava_linux_amd64.tar.gz'
+
+install_lava() {
+	local version=$1
+
+	local url
+	if [[ $version == 'latest' ]]; then
+		url=$GH_DOWNLOAD_URL_LATEST
+	else
+		url=$(printf "${GH_DOWNLOAD_URL_FMT}" "${version}")
+	fi
+
+	# Make sure the install directory exists.
+	install_dir="/tmp/bin"
+	mkdir -p "${install_dir}"
+
+	# Try to download a Lava release from GitHub.
+	if (curl -LsSf "${url}" | tar -xz -C "${install_dir}" lava) 2> /dev/null; then
+		return 0
+	fi
+}
+
+# Install Lava.
+if ! install_lava "${LAVA_VERSION}"; then
+	echo 'error: could not install lava' >&2
+	exit 1
+fi
+
+for row in $(cat pages.json | jq --compact-output '.[]'); do
+	_jq() {
+		echo ${row} | jq --raw-output ${1}
+	}
+	topic=$(_jq '.topic')
+	file=$(_jq '.file')
+	header=$(_jq '.header')
+
+	dir=$(dirname ${file})
+
+	mkdir -p ${dir}
+  ${install_dir}/lava help ${topic} > ${file}
+  fix_headers ${file}
+  add_main_header ${file} ${header}
+done
+
+curl -sSL https://github.com/rust-lang/mdBook/releases/download/v0.4.37/mdbook-v0.4.37-x86_64-unknown-linux-gnu.tar.gz | tar -xz --directory=${install_dir}
+check_status $? "error: could not install mdBook"
+
+sudo chmod +x ${install_dir}/mdbook
```

Comment:
Move to a function `install_mdbook` similar to `install_lava`. Also, move the mdbook version to an env var at the beginning of the script. Finally, I would download it with:

```
curl -LsSf "https://github.com/rust-lang/mdBook/releases/download/${MDBOOK_VERSION}/mdbook-${MDBOOK_VERSION}-x86_64-unknown-linux-gnu.tar.gz" | tar -xz -C "${install_dir}" mdbook
```